src/common/debug.c

```diff
@@ -41,16 +40,27 @@
 
 # include "debug_priv.h"
 
+#include "debug.h"
+#include "_error.h"
+
 #ifndef PATH_SEPARATOR
 # define PATH_SEPARATOR '/'
 #endif
 
+static const struct oscap_string_map OSCAP_VERBOSITY_LEVELS[] = {
+    {DBG_E, "ERROR"},
+    {DBG_W, "WARNING"},
+    {DBG_I, "INFO"},
+    {DBG_D, "DEVEL"},
+    {DBG_UNKNOWN, NULL}
+};
+
 #  if defined(OSCAP_THREAD_SAFE)
 #   include <pthread.h>
 static pthread_mutex_t __debuglog_mutex = PTHREAD_MUTEX_INITIALIZER;
 #  endif
 static FILE *__debuglog_fp = NULL;
-int __debuglog_level  = -1;
+int __debuglog_level = DBG_UNKNOWN;
```

Comment:
It is because it is declared as `extern` in the header file. But after I have removed macro `debug(l)`, it is no longer needed to declare it as `extern`. Nobody uses it now. I think I can remove that declaration from header file, and add `static` modifier to it, because it is used only inside this module.